OVP/D3D9Client/D3D9Util.h

```diff
@@ -71,8 +71,13 @@ const char *_PTR(const void *p);
 
 // helper function to get address of a temporary
 // NB: use with caution
-template<typename T>
-T* ptr(T&& x) { return &x; }
+
+
+// Required only with c++20 without /permissive flag
+// template<typename T>
+// T* ptr(T&& x) { return &x; }
+
+#define ptr &	// use for faster code
```

Comment:
It's not faster. With /O2 flag MSVC produces identical code for both versions: 

https://godbolt.org/z/jnqvMq4G3

Except this is a valid code, as long as you use the address within the same expression.